plugins/chat/app/controllers/chat/api/channels_controller.rb

```diff
@@ -68,21 +68,23 @@ def create
   end
 
   def show
-    if params[:target_message_id].present? || params[:include_messages].present? ||
-         params[:fetch_from_last_read].present?
+    if params[:target_message_id].present? || params[:target_date].present? ||
```

Comment:
Since this adds a new parameter to the controller, it would be good to describe how to use this, maybe in the PR description.